src/engine.py

```
@@ -15755,44 +15759,105 @@ async def run_turn(transcript_text: str) -> None:
+                                                                    tool_call_id=next_function_call_id,
+                                                                    tool_name=next_name,
+                                                                    canonical_name=tool_registry.canonicalize_tool_name(next_name),
+                                                                    parameters=next_args,
+                                                                    result={
+                                                                        "status": "blocked",
+                                                                        "message": f"Tool '{next_name}' not allowed for this call",
+                                                                    },
+                                                                )
                                                                 continue
                                                         except Exception:
                                                             pass
```

Ruff (0.16.0)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)